health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/AdvancedMapCharts/HeatMap/Map.js

```
@@ -0,0 +1,369 @@
+
+      return (
+        <button
+          className={"digit-heat-map-zoom-button"}
+          onClick={() => {
+            if (label === "+") {
+              handleZoomIn();
+            } else {
+              handleZoomOut();
+            }
+          }}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)